src/userInterface/panels/SideNavPanel/SideNavPanel.module.scss

```diff
@@ -1,12 +1,10 @@
-.jsonConfigContainer {
-    margin: 10px 0;
+:root {
```

Comment:
no need for css variables use sass variables https://sass-lang.com/documentation/variables

Reply:
since these variables are not set dynamically

Reply:
fixed